css/components/header.css

```diff
@@ -1,81 +1,64 @@
 .header {
-    background-image:url('/design/images/bg-hero-desktop.svg'); 
-    padding: 60px 120px;
-    background-color: var(--pale-cyan);
+  background-image: url('/design/images/bg-hero-desktop.svg');
+  padding: 60px 120px;
+  background-color: var(--pale-cyan);
 }
 
 .header__top-container {
```

Comment:
I think here it is an element of your header block (BEM), with the other one it is a new block you are starting.